### DISCUSSION TITLE
Tests title in --steps option looks not good in console

### DISCUSSION THREAD

**Discussion by javigomez** [Codeception/Codeception#6178] [Ideas]
when you run codecept like this: `vendor/bin/codecept run --steps acceptance`

All the steps are printed in the console. But if you check the Test title is cut:

``` php
    public function administratorArchiveCategory(\Step\Acceptance\category $I)
    {
        $I->am('Administrator');
        $I->wantToTest('Archiving a Category in /administrator/');
        $I->doAdministratorLogin();
```

ends in (see the pink text):
<img width="923" alt="screen shot 2015-11-01 at 10 21 42" src="https://cloud.githubusercontent.com/assets/1375475/10868068/61d4b318-8082-11e5-9280-c893ca57b9d8.png">

and:

``` php
    public function administratorCreateCategoryWithoutTitleFails(AcceptanceTester $I)
    {
        $I->am('Administrator');
        $I->wantToTest('Category creation in /administrator/ without title fails');

```

<img width="890" alt="screen shot 2015-11-01 at 10 21 26" src="https://cloud.githubusercontent.com/assets/1375475/10868070/66a5271a-8082-11e5-88c7-6e1c35c17110.png">

I would suggest to break the text in two lines:

```
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
The text in the function ->IwantToText
(TheTestNameCest::TheTestFunction)
Scenario:
* As an Administrator
* I do administrator login 
```

That way there will be more room for the text in iWantToText if is long.
How does it sound?

modifying the vendor/codeception/codeception/src/Codeception/Scenario.php should do the trick.

**Comment by DavertMik** [comment]
Thanks for the idea. I will try to make it implemented

And when `--steps` is not provided? This ugly cut still will be there. 
Maybe we should cut filename instead, how do you think @javigomez?
